### PR TITLE
Add compound data types exe

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Review by sleviim — COMMENTED** [review]
Just to be sure - was this one tested?

File: docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Let's Practice Together \#1 
+
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md:2]: Remove this extra line and the presentation will work correctly (with the background on the first slide)

File: docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+- Create a file named *my_list.py*
+- Create a new list with the following items: "p","k","p" ,"p","4","J","p"
+- Append "b" to the list you created
+- Remove item in index 1
+- Replace all "p" values with "o"
+- Replace "4" value with "d"
+- Replace the first item with "G"
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md:10]: Please add the image at the end
<div style="text-align:center">

File: docs/course_materials/exercises/04_Compound_data_types/class_exercise_2.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+### Let's Practice Together \#2
+
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/class_exercise_2.md:2]: same - remove this line and add image

File: docs/course_materials/exercises/04_Compound_data_types/class_exercise_3.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  homework:  90.0,97.0,75.0,92.0
+  quizzes: 88.0,40.0,94.0
+  tests: 75.0,90.0
+  Add those scores to the student dictionary 
+- Itzik  was given the opportunity to change his lowest test score, he re-tested and got 95.0. please change his score.
+- Print: 
+    - The student's name
+    - The student's homework
+    - The student's quizzes
+    - The student's tests
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/class_exercise_3.md:16]: add image, please

File: docs/course_materials/exercises/04_Compound_data_types/homework.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+
+Before each exercise, write in a comment the exercise number, for example:
+
+    # 1 - empty list 
+    list1 = []
+    list2 = list()
+    # 2 
+
+#### Create a list
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/homework.md:26]: add 1 line after headline

File: docs/course_materials/exercises/04_Compound_data_types/homework.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+#### Create a list 
+1. Create two empty lists and named them  list1 and list2, create those empty lists in two different ways.
+2. Create a list with the following numbers: 2,3,4,5,6 the list name should be l1
+3. print the index of the number 6? 
+4. Append two numbers to the list 7 and 8
+5. Print the length of the list
+6. Add the number 1 to the list, 1 should be the first item on the list
+
+#### Slicing a list
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/homework.md:34]: same

File: docs/course_materials/exercises/04_Compound_data_types/homework.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+1. Print first 4 numbers of l1. 
+2. Print 4 last numbers of l1. 
+3. Print the two numbers in the middle
+4. Print l1[20], which error did you get? What does this error say? ( write your answer in a comment #)
+	
+#### Update the list
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/homework.md:40]: same

File: docs/course_materials/exercises/04_Compound_data_types/homework.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+2. Create a new list - l2, with the items: -1, -2 ,-3 , print the list  
+3. Create a new list l3 with the items of l1 and l2, print the list.
+4. Sort l3 list, print the sorted list
+5. Print the length of l1, l2, and l3
+
+
+## dictionary
+
+
+### countries.py
```
Comment (sleviim) [inline docs/course_materials/exercises/04_Compound_data_types/homework.md:51]: same

**Review by sleviim — CHANGES_REQUESTED** [review]

**Comment by sleviim** [comment]
Please refer to the other comments as well (add the image at the end of the exe. slides + add 1 line between the headline and the text of the homeworks exe.)

**Comment by sleviim** [comment]
Also, please resolve the conflicts so it can be merged

File: docs/course_materials/exercises/04_Compound_data_types/class_exercise_3.md (hunk 4 above)
Comment (shiramax) [reply docs/course_materials/exercises/04_Compound_data_types/class_exercise_3.md:16]: which image?

Comment (sleviim) [reply docs/course_materials/exercises/04_Compound_data_types/class_exercise_3.md:16]: &lt<div style="text-align:center">&gt

**Review by sleviim — APPROVED** [review]
LGTM
Sorry for all the mess

**Pull request merged**